### PR TITLE
SS-698 set default time zone to stockholm time

### PR DESCRIPTION
**Pull request by anondo1969**
## Description

Source: https://scilifelab.atlassian.net/browse/SS-698

1. Set default time zone to Stockholm time.
2. Mention it as tooltip info besides app info in /projects page.

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help!
This is simply a reminder of what we are going to look for before merging your code._

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [x] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ ] I have updated the related documentation (if necessary)
- [x] I have added a reviewer for this pull request
- [x] I have added myself as an author for this pull request
- [x] In the case I have modified settings.py, then I have also updated the studio-settings-configmap.yaml file in serve-charts

## Further comments

Anything else you think we should know before merging your code!

### CODE REVIEW

**Comment by anondo1969** [comment]
Need to update the studio-settings-configmap.yaml file in serve-charts beacuse of the change in settings.py file.

**Comment by anondo1969** [comment]
> Need to update the studio-settings-configmap.yaml file in serve-charts beacuse of the change in settings.py file.

Done.

File: templates/projects/partials/app_instances_table.html [hunk 1]
```diff
@@ -40,7 +39,11 @@
             {% else %}
             <td class="sorting_1">{{ instance.name }}</td>
             {% endif %}
-            <td class="d-none d-xxl-table-cell">{{ instance.created_on | date:"d/n/y H:i" }}</td>
+            <td class="d-none d-xxl-table-cell">{{ instance.created_on | date:"d/n/y H:i" }}
+                <span
+                class="bi bi-clock" data-bs-toggle="tooltip" data-bs-placement="right" data-bs-original-title="{{ objs.timezone }}">
+                </span>
+            </td>
```
[the remark this answers is not in the document]
Comment (anondo1969) [reply templates/projects/partials/app_instances_table.html:46]: Thanks Nikita. This looks nice, clear, and compact. Shall I implement this one?

[the remark this answers is not in the document]
Comment (churnikov) [reply templates/projects/partials/app_instances_table.html:46]: Well, if you like it and have time:)

Otherwise, the current one it also good.

**Pull request merged**